app/views/shared/metadata/edit/_study.html.erb

```diff
@@ -59,22 +59,22 @@
       %>
 
       <% metadata_fields.related_fields(to: :data_release_strategy, when: Study::DATA_RELEASE_STRATEGY_NOT_APPLICABLE) do %>
-        <%= group.select(:data_release_prevention_reason, [''] + Study::DATA_RELEASE_PREVENTION_REASONS) %>
+        <%= group.select(:data_release_prevention_reason, study.data_release_prevention_options) %>
         <%= group.text_area(:data_release_prevention_approval) %>
         <%= group.text_area(:data_release_prevention_reason_comment) %>
       <% end %>
       <% metadata_fields.related_fields(to: :data_release_strategy, in: Study::DATA_RELEASE_STRATEGIES, not: Study::DATA_RELEASE_STRATEGY_NOT_APPLICABLE) do %>
         <% metadata_fields.related_fields(to: :data_release_timing, when: Study::DATA_RELEASE_TIMING_DELAYED) do %>
-          <%= group.select(:data_release_delay_reason, [''] + Study::DATA_RELEASE_DELAY_REASONS_STANDARD) %>
+          <%= group.select(:data_release_delay_reason, study.data_release_delay_options) %>
           <%
             group.change_select_options_for(:data_release_delay_reason, when: :data_release_study_type_id, values: {
-              DataReleaseStudyType.assay_types.map(&:id)     => [ '' ] + Study::DATA_RELEASE_DELAY_REASONS_ASSAY,
-              DataReleaseStudyType.non_assay_types.map(&:id) => [ '' ] + Study::DATA_RELEASE_DELAY_REASONS_STANDARD
+              DataReleaseStudyType.assay_types.map(&:id)     => [ '' ] + study.data_release_delay_options(assay_option: true),
+              DataReleaseStudyType.non_assay_types.map(&:id) => [ '' ] + study.data_release_delay_options
             })
           %>
 
           <%= group.select(:data_release_delay_period, Study::DATA_RELEASE_DELAY_PERIODS) %>
-          <% metadata_fields.related_fields(to: :data_release_delay_reason, when: Study::DATA_RELEASE_DELAY_FOR_OTHER) do %>
+          <% metadata_fields.related_fields(to: :data_release_delay_reason, in: [Study::DATA_RELEASE_DELAY_FOR_OTHER, 'other']) do %>
```

Comment:
This is a bit ugly, but we now have two 'other' options (old and new) and we need to show some fields conditionally based on this option.